news/2021-09-16-osu-world-cup-2021-registrations-now-open.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "osu! World Cup 2021: Registrations now open!"
+date: 2021-09-16 09:00:00 +0000
+---
+
+With 2021 coming to an end, it's time to sign off with the final tournament of the year. The osu! World Cup is back and better than ever, with registrations now open!
+
+![](/wiki/shared/news/2021-09-16-osu-world-cup-2021-registrations-now-open/owc-2021-banner.jpg)
+
+For more details, refer to the **[osu! World Cup 2021 wiki article](/wiki/Tournaments/OWC/2021)**. You can discuss this event in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1420416)** or at the public **[community server](https://discord.gg/0Vxo9AsejDkGlk3H)**.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | Sep 16 – Sep 30 |
+| Qualifier showcase | Oct 09 (14:00 UTC) |
+| Qualifier stage | Oct 16 & Oct 17 |
+| Round of 32 | Oct 23 & Oct 24 |
+| Round of 16 | Oct 30 & Oct 31 |
+| Quarterfinals | Nov 06 & Nov 07 |
+| Semifinals | Nov 13 & Nov 14 |
+| Finals | Nov 20 & Nov 21 |
+| Grand Finals | Nov 27 & Nov 28 |
+
+## Prizes
+
+The osu! World Cup 2021 offers a $5,000 prize pool, along with limited edition goods!
+
+- **1st:** 50% of the prize pool, exclusive limited-edition merch, profile badge, "osu! Champion" user title for one year
+- **2nd:** 34% of the prize pool, exclusive limited-edition merch, profile badge
+- **3rd:** 16% of the prize pool, exclusive limited-edition merch, profile badge
+
+## How do I register?
+
+- **[Click here!](https://osu.ppy.sh/community/tournaments/30)**
+- Teams will be formed by each country's captain *after* the registration phase has concluded.
+- **The deadline for registering is September 30, 23:59 UTC. Make sure to get yours in before then!**
+
+*Notice: you may only participate if your osu! Global Ranking is #5000 or higher, and you have not violated the [osu! community rules](/wiki/Rules) in the past 12 months*.
+
+If your registration passes our internal checks, you will be placed on the candidate list for your country. When that country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.
+
+*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of time to avoid any last-minute headaches. If you run into any issues, send an email to [tournaments@ppy.sh](mailto:accounts@ppy.sh).*
```

Comment:
```suggestion
*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of time to avoid any last-minute headaches. If you run into any issues, send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).*
```
👀

Reply:
>tournament guy forgets about the tournaments email